config/cmake/config/vxlGetCXXCompilerVersion.cxx

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+
+int main()
+{
+  #if __cplusplus >= VXL_CXX_TEST_VERSION
+    std::cout << __cplusplus << ">=" << VXL_CXX_TEST_VERSION << std::endl;
+    return 0;
```

Comment:
Trivial point, but I know in ITK the `EXIT_FAILURE`/`EXIT_SUCCESS` macros are preferred--is the same true in VXL?